SinghxRaj.CodingTracker/Display.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace SinghxRaj.CodingTracker;
+
+internal class Display
+{
+    internal static void PrintIntro()
+    {
+        Console.WriteLine("Welcome to Coding Tracker!");
+        Console.WriteLine("With Coding Tracker, you can " +
+                    "keep track of all your coding sessions.");
+        Console.WriteLine();
+    }
+
+    internal static void PrintOptions()
+    {
+        Console.WriteLine("Type 0 - Exit application");
```

Comment:
Users should also be able to delete/update data, as per requirements: 
![image](https://user-images.githubusercontent.com/108711476/213915528-0c9bcb4d-b276-4df0-ac8c-34927d0910d7.png)

From the habit tracker project. 
![image](https://user-images.githubusercontent.com/108711476/213915506-847b335a-048a-4322-8f44-ef879bb6f9cd.png)

Reply:
Added a delete coding session feature and updating coding session feature.